### PR TITLE
Rename advertisement to liqo controller manager

### PR DESCRIPTION
**Pull request by alacuku**
# Description

This PR renames the `advertisement-operator `to `liqo-controller-manager`.

### CODE REVIEW

File: build/liqo-controller-manager/Dockerfile [hunk 1]
```diff
@@ -6,9 +6,9 @@ COPY go.mod ./go.mod
 COPY go.sum ./go.sum
```
Comment (palexster) [inline build/liqo-controller-manager/Dockerfile:6]: I think this file has been dropped.

**Review by aleoli — APPROVED** [review]

**Review by palexster — APPROVED** [review]

**Pull request merged**